requirements.in

```diff
@@ -14,4 +14,4 @@ redis==2.10.6
 requests
 uwsgi
 djangorestframework
-
+ocw-data-parser
```

Comment:
Can you change to `ocw-data-parser==0.0.5` and then run `pip-compile --output-file requirements.txt requirements.in` to auto-update `requirements.txt`

Reply:
Done. Will merge once test pass.